exasol_integration_test_docker_environment/lib/base/base_task.py

```diff
@@ -107,13 +110,16 @@ def get_output(self) -> Any:
         return self._outputs_cache
 
 
+BaseTaskType = TypeVar("BaseTaskType", bound="BaseTask")
```

Comment:
Explain the purpose of the use of generics with run_dependencies

Reply:
Done